.github/pull_request_template.md

```diff
@@ -9,11 +9,16 @@
 <!--- Why is this change required? What problem does it solve? -->
 <!--- If it fixes an open issue, please link to the issue here. -->
 
```

Comment:
Should we add the Fixes # tag? And make a comment to delete it if they do not want it?